flow.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   plugins: [
-    require('babel-plugin-syntax-flow'),
-    require('babel-plugin-transform-flow-strip-types'),
+    require.resolve('babel-plugin-syntax-flow'),
```

Comment:
`require.resolve` gets the absolute path so that libs are loaded only when they're needed?